docs/Containers/Container-Tasks.md

```diff
@@ -118,6 +118,52 @@ The following is the process by which you can create a usable image from your cu
     Evergreen YAML must also use a **&lt;SHA&gt;** tag, which would be the hash
     of the corresponding commit in our [image repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
 
+The following is a starter Dockerfile that abides by our image policy and
+includes commonly used tools and packages (e.g. Go, Python, NodeJS, etc.)
+that you can copy/paste and use as a reference for creating your own. 
```

Comment:
I would add a note that this is just a template to get started, and straight copy-pasting of the Dockerfile is not recommended since we won't be updating this template. These packages will go out of date, and users should use supported releases of their software dependencies as part of best practices rather than let users assume the dependencies we're listing here are the most up-to-date and supported ones.